doc/source/examples/sharp-immersed-boundary/flow-around-particle-sharp/flow-around-particle-sharp.rst

```diff
@@ -0,0 +1,218 @@
+        subsection bc 2
+            set id   = 3
+            set type = slip
+        end
+        subsection bc 3
+            set id   = 4
+            set type = slip
+        end
+        subsection bc 4
+            set id   = 5
+            set type = slip
+        end
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+In order to produce a varying Reynolds number, we vary the kinematic viscosity of the fluid. We define the Reynolds number as :math:`Re = \frac{UD}{\nu}` where :math:`U` and :math:`D` are fixed at 1. Therefore, 
+the kinematic viscosity is defined as :math:`\nu = \frac{1}{Re}`. 
+
+Note that this definition of the Reynolds number depends only on the particle's diameter, not its length. We select this parameter because it primarily governs the flow regime around the particle.
+The following block defines the viscosity for a Reynolds number of 1:
+
+.. code-block:: text
+
+    subsection physical properties
+        subsection fluid 0
+            set kinematic viscosity = 1
+        end
+    end
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~
+
+With the Reynolds number ranging from 0.01 to 5, the simulation can easily be solved by using a steady-state solver. The mesh is sufficiently refined to capture the flow around the particle and therefore doesn't 
+require any mesh adaptation.
+
+.. code-block:: text
+
+    subsection simulation control
+        set method            = steady
+        set output path       = ./particle_drag/
+    end
+
+---------------
+Results
+---------------
+
+We obtain the following solution for the velocity and pressure:
+
+Velocity:
+ 
+.. image:: images/flow-around-particle-sharp-velocity.png
+    :alt: Velocity field around the particle
+    :align: center
+
+Pressure: 
+
+.. image:: images/flow-around-particle-sharp-pressure.png
+    :alt: Pressure field around the particle
+    :align: center
+
+The drag coefficient :math:`C_d` is:
+
+.. math::
+
+    C_d = \frac{F_d}{0.5 \rho U^2 A} = \frac{2f_x}{U^2 L D}
+
+Where :math:`F_d` is the drag force, :math:`\rho` is the fluid density, :math:`U` is the velocity, and :math:`A` is the projected area :math:`(A = LD)`. In Lethe, we solve for :math:`f = F/\rho`. Therefore, the drag force is calculated as the force applied 
+on the particle in the flow direction using :math:`f_x` from the simulation output.
+
+The following graph shows the drag coefficient as a function of the Reynolds number. The results are compared to the literature values for a perpendicular cylinder with an aspect ratio of 2.
+
+.. image:: images/flow-around-particle-sharp-CD-Re-study.png
+    :alt: Drag coefficient as a function of Reynolds number
+    :align: center
+
+Here we show the drag models of Vakil and Green [1] and Hölzer and Sommerfeld [2] for comparison. Vakil's drag model is specifically made for cylinders in flow of Reynolds from 1 to 40, whereas Hölzer's model is meant to be more general for non-spherical particles defined by their sphericity in all flow regimes.
```

Comment:
Please add the references at the bottom of the example, see others examples to see how it is done.